Update README.md

shiliyu opened the pull request:
Update README.md just being much ado about nothing.

shiliyu committed:
Update README.md

Update README.md

blindpirate commented on README.md:
```diff
@@ -1,8 +1,10 @@
+## A replacement for deprecated Gradle 5 osgi plugin
 
-Since Gradle 6, `osgi` plugin was removed from Gradle core plugins, but some builds still depend on this plugin.
+`OSGI` plugin has been removed from Gradle core plugins since Gradle 6, yet some builds still depend on it.
```
```suggestion
`osgi` plugin has been removed from Gradle core plugins since Gradle 6, yet some builds still depend on it.
```

blindpirate commented on README.md:
```diff
@@ -1,8 +1,10 @@
+`OSGI` plugin has been removed from Gradle core plugins since Gradle 6, yet some builds still depend on it.
 
-This repo extracts the legacy `osgi` plugin (and unit test) from Gradle 5.6.4 source code.
+This repo extracts the legacy `OSGI` plugin (and unit test) from Gradle 5.6.4 source code.
```
```suggestion
This repo extracts the legacy `osgi` plugin (and unit test) from Gradle 5.6.4 source code.
```

blindpirate commented on README.md:
```diff
@@ -1,8 +1,10 @@
-This repo extracts the legacy `osgi` plugin (and unit test) from Gradle 5.6.4 source code.
+This repo extracts the legacy `OSGI` plugin (and unit test) from Gradle 5.6.4 source code.
+
+If you're still using the deprecated `OSGI` Gradle plugin ('osgi'), you need to migrate to using ('com.github.blindpirate.osgi').
```
```suggestion
If you're still using the deprecated `osgi` Gradle plugin in Gradle 6, you can use this plugin (`com.github.blindpirate.osgi`) as an alternative.
```

blindpirate reviewed (commented):
I know it's written as `OSGI` in the documentation, but I prefer `osgi` since it's more developer-friendly, we usually write our build script as:

```
apply plugin: 'osgi'
```

Also, this plugin is not intended to be a replacement - it's a workaround. So I wouldn't say "you need to migrate to this plugin".

shiliyu committed:
Update README.md

Co-Authored-By: Bo Zhang <bo@gradle.com>

shiliyu committed:
Update README.md

Co-Authored-By: Bo Zhang <bo@gradle.com>

shiliyu committed:
Refine README.md

Change the choice of words. Emphasize  (`com.github.blindpirate.osgi`) is just better option for use in Gradle 6 and above than the deprecated one.

blindpirate commented:
Thanks @shiliyu
